src/Components/Authentication/Login/Login.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useAuthProvider } from "../../../Context/AuthContext/AuthProvider";
+export const Login = () => {
```

Comment:
Add empty lines between two logical blocks 

Like here, between import and the component there should be an empty line indicating the separation of code.